code/datums/components/afterimage.dm

```diff
@@ -31,16 +31,15 @@
 
 /datum/component/after_image/RegisterWithParent()
 	loop_timer = addtimer(CALLBACK(src, PROC_REF(spawn_image)), rest_time, TIMER_LOOP|TIMER_UNIQUE|TIMER_STOPPABLE)//start loop
-	RegisterSignal(parent, COMSIG_MOB_CLIENT_PRE_MOVE, PROC_REF(update_step))
-	RegisterSignal(parent, COMSIG_MOB_CLIENT_MOVED, PROC_REF(update_glide))
+	RegisterSignal(parent, COMSIG_MOVABLE_PRE_MOVE, PROC_REF(update_step))
+	RegisterSignal(parent, COMSIG_MOVABLE_MOVED, PROC_REF(update_glide))
```

Comment:
Make sure this doesnt fuck up the sandevistan visual effect
I remember having to do some fucky stuff for it to work

Reply:
It still worked when I tested it earlier.